### PR TITLE
AARA_546_create_contacts_form implementation

### PR DESCRIPTION
**Pull request by lauherkcambria**
#### Which User story does this relate to (link)?
-https://osi-cwds.atlassian.net/browse/AARA-546

#### Brief feature description
- new/ edit contact form impl

#### Does this PR block/effect other stories?
- na/

#### Do any commands needs to be run in order to run the project with this PR?
-

#### Code coverage percent diff as a result of this PR?
-

### CODE REVIEW

**Comment by LeonidMarushevskyi** [comment]
Jenkins please rebuild this

**Comment by LeonidMarushevskyi** [comment]
Jenkins retest this please

File: app/javascript/contacts/contact_edit_view/index.js [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+                            label={'Method of Contact' + RfaCommon.requiredIndicator}
+                            onChange={(event) => this.setContactState('contact_method', dictionaryNilSelect(event.target.options))} />
+                        </div>
+                      </div>
+                      <InPersonData
+                        inPersonContactData={contact.in_person_contact_data}
+                        setInPersonData={this.setInPersonData}
+                        contactLocations={this.props.contact_locations}
+                        contactVisitTypes={this.props.contact_visit_types} />
+                      <div className='row' >
```
Comment (nasar-cambria) [inline app/javascript/contacts/contact_edit_view/index.js]: Can you indent title column. It's not aligned with other fields. Same with Notes Column

File: app/javascript/contacts/contact_edit_view/inPersonData.js [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+        <div className='row' >
+          <div className='col-xs-4' >
+            <YesNoRadioComponent
+              gridClassName='col-xs-6'
+              label='Collateral Visit?'
+              idPrefix='collateralVisit'
+              value={inPersonContactData.is_collateral_visit}
+              onFieldChange={(event) => { this.props.setInPersonData('is_collateral_visit', event.target.value === 'true') }} />
+          </div>
+          <div className='col-xs-2' />
```
Comment (nasar-cambria) [inline app/javascript/contacts/contact_edit_view/inPersonData.js]: What is this extra ```div``` for ?

File: app/javascript/contacts/contact_edit_view/index.js [hunk 3]
```diff
@@ -0,0 +1,182 @@
...
+                      </div>
+                      <div className='row' >
+                        <div className='col-xs-4' >
+                          <DateField
+                            id='dateOfContact'
+                            label={'Date' + RfaCommon.requiredIndicator}
+                            value={FormatDateForDisplay(contact.date)}
+                            onChange={(event) => this.setContactState('date', FormatDateForPersistance(event.target.value))} />
+                        </div>
+                        <div className='col-xs-2' />
```
Comment (nasar-cambria) [inline app/javascript/contacts/contact_edit_view/index.js]: If we are using an extra div for space. Can we use css property to align the columns. ```float: right;
    margin-right: 14rem;``` should work

File: app/javascript/contacts/contacts_list/ContactsTable.jsx [hunk 4]
```diff
@@ -34,43 +36,48 @@ export default class ContactsTable extends React.Component {
...
+      headerClassName: 'contacts-th',
+      id: 'contact',
+      sortable: false,
+      accessor: d => d,
+      maxWidth: 600,
+      className: 'contacts-td',
+      Cell: row => (<a href={urlPrefixHelper('/rfa/a01/' + rfa01aId + '/contacts/' + row.value.id + '/edit')}>{row.value.title}</a>)
+      // Cell: (row, rfa01aId) => {
+      //   return (<a href={urlPrefixHelper('/rfa/a01/' + rfa01aId + '/contacts/' + row.value.id + '/edit')}>{row.value.title}</a>)
+      // }
```
Comment (nasar-cambria) [inline app/javascript/contacts/contacts_list/ContactsTable.jsx]: do we need this here?

**Pull request merged**